watertap/examples/flowsheets/RO_with_energy_recovery/multi_sweep.py

```diff
@@ -0,0 +1,112 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+from watertap.tools.parameter_sweep import PredeterminedFixedSample, parameter_sweep
+import watertap.examples.flowsheets.RO_with_energy_recovery.RO_with_energy_recovery as RO
+from watertap.examples.flowsheets.RO_with_energy_recovery.RO_with_energy_recovery import (
+    ERDtype,
+)
+
+
+def set_up_sensitivity():
+    outputs = {}
+
+    m = RO.build(erd_type=ERDtype.pump_as_turbine)
+    RO.set_operating_conditions(m)
+    RO.initialize_system(m)
+    RO.solve(m)
+    m.fs.feed.properties[0].flow_mass_phase_comp.unfix()
+    m.fs.feed.properties[0].flow_vol_phase["Liq"].fix()
+    m.fs.feed.properties[0].conc_mass_phase_comp["Liq", "NaCl"].fix()
+    RO.optimize_set_up(m)
+    m.fs.eq_minimum_water_flux.deactivate()
```

Comment:
What happens if we remove this deactivate line? I wonder if this was something that we thought would help solution convergence across different conditions but ended up not being necessary. Can you quickly double-check whether tests fail or solutions no longer converge for cases (which I assume would be part of your testing anyhow)?

If it is indeed necessary, let's keep it; if not, let's remove this line

Reply:
Get this warning when that line is removed:
`WARNING: Loading a SolverResults object with a warning status into
model.name="unknown";
    - termination condition: maxIterations
    - message from solver: Ipopt 3.13.2\x3a Maximum Number of Iterations
      Exceeded.`

So I'll keep it in.